src/utils/WagmiProvider.tsx

```diff
@@ -19,37 +17,38 @@ const chains = defaultChains
 
 // Add local node support if developing
 if (process.env.NODE_ENV === "development") {
-  chains.push(...developmentChains)
+  chains.push(chain.hardhat)
+  chains.push(chain.localhost)
 }
 
-// Set up connectors
-const connectors = ({ chainId }: { chainId?: number | undefined }) => {
-  return [
-    new InjectedConnector({ chains }),
-    new WalletConnectConnector({
-      chains,
-      options: {
-        qrcode: true,
-        chainId: networkId,
-        rpc: {
-          [networkId]: alchemyApiUrlHttp,
-        },
-      },
+const { provider, webSocketProvider } = configureChains(chains, [
+  alchemyProvider({ alchemyId: alchemyApiKey }),
+  publicProvider(),
+  jsonRpcProvider({
+    rpc: () => ({
+      http: "http://127.0.0.1:8545",
```

Comment:
Does this fix now the annoying errors popping up when running against a local node?

Reply:
Yeah, I'm not getting those messages anymore 🙏🏽